spec/dor/identifiable_spec.rb

```diff
@@ -312,4 +312,14 @@ class IdentifiableItem < ActiveFedora::Base
       expect(item.get_related_obj_display_title(mock_apo_obj, mock_default_title)).to eq(mock_default_title)
     end
   end
+
+  describe '#adapt_to_cmodel' do
+    context 'for a Hydrus collection' do
```

Comment:
i think it makes sense to add tests for an item adaption for clarity. item_druid_bb004bn8654.xml is a fixture for a Hydrus item and item_druid_dc235vd9662.xml is a fixture for a Dor item.